style.css

```diff
@@ -159,18 +172,71 @@ body {
   top: 80px;
   left: 25px;
   right: 60px;
-  font-size: 13px;
   font-size: 16px;
   font-family: tahoma, sans-serif;
-  color: #ffffff;
 }
 
 .smallText a {
   text-decoration: none;
+  color: #ffffff;
 }
 
 .smallText a:hover {
   text-decoration: underline;
 }
 
+#smallArticles {
```

Comment:
It is always a bad idea to stuck some style to an ID. With this approach, the style is not reusable anymore. Why do not use just a regular CSS class?